### PR TITLE
Update documents to notice the announcement about maintenance phase

### LINKED ISSUE #8513: Issue management after v7.0.0 release
As we [announced](https://chainer.org/announcement/2019/12/05/released-v7.html), Chainer will go on to the maintenance phase.

- [ ] Update docs to announce it: README, contribution guide, pull-request template
- [x] Remove `contribution-wanted` label from issues (done: 2019/12/17)
- [x] Reprioritize existing issues

### PR DESCRIPTION
**Pull request by himkt**
This PR update documents (README, PR template, contribution guide) to
notify users about the announcement of future Chainer development.

Close #8513

### CODE REVIEW

**Comment by himkt** [comment]
I made draft updates. Could someone take a look?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
+**Chainer will go on to the maintainance phase, further development will be limited to bug-fixes and maintenance.**
```
Comment (emcastillo) [inline .github/PULL_REQUEST_TEMPLATE.md]: typo on maintenance.

File: README.md [hunk 2]
```diff
@@ -54,10 +54,10 @@ $ nvidia-docker run -it chainer/chainer /bin/bash
 ```
 
 
-## Contribution
+## Maintainance
```
Comment (emcastillo) [inline README.md]: same `maintainance` -> `maintenance`

File: README.md [hunk 3]
```diff
@@ -54,10 +54,10 @@ $ nvidia-docker run -it chainer/chainer /bin/bash
...
 
-Any contributions to Chainer are welcome!
-If you want to file an issue or send a pull request, [please follow the contribution guide](https://docs.chainer.org/en/stable/contribution.html).
+As we [announced](https://chainer.org/announcement/2019/12/05/released-v7.html), Chainer  will go on to the maintainance phase.
```
Comment (emcastillo) [inline README.md]: ditto

**Comment by himkt** [comment]
Oh sorry, I fixed. :bow: @emcastillo

**Comment by kmaehashi** [comment]
As there're some more files I think we need to modify, I raised another PR. Thanks again for the contribution!

**Pull request closed without merging**